cmd/worker/workercmd/cmd.go

```diff
@@ -75,11 +77,47 @@ func (m *Main) Run(ctx context.Context) error {
 		return err
 	}
 
+	var ps persistence.Service
+	if m.cfg.CheckDuplicates {
+		sqlDB, err := persistence.Open(m.cfg.Persistence.Driver, m.cfg.Persistence.DSN)
+		if err != nil {
+			m.logger.Error(err, "Error initializing database pool.")
+			return err
+		}
+		m.dbClient = db.NewClient(
+			db.Driver(
+				sqlcomment.NewDriver(
+					sql.OpenDB(m.cfg.Persistence.Driver, sqlDB),
+					sqlcomment.WithDriverVerTag(),
+					sqlcomment.WithTags(sqlcomment.Tags{
+						sqlcomment.KeyApplication: Name,
+					}),
+				),
+			),
+		)
+		if m.cfg.Persistence.Migrate {
+			err = m.dbClient.Schema.Create(ctx)
+			if err != nil {
+				m.logger.Error(err, "Error migrating database.")
+				return err
+			}
+		}
+		ps = entclient.New(m.dbClient)
+	}
```

Comment:
Just moved the previous code from below and added the `ps` variable declaration and assignment.